### PR TITLE
Add Option to select Branch for Plugin Installation

### PR DESCRIPTION
**Pull request by mxub**
Often when I want to install a new Plugin it has no `package.json` so I want to use the following workflow:
1. Fork Plugin project and add the `package.json`
2. Create a pull request
3. Install Plugin via CLI using my fork

---

It seems like Github wants you to create a new branch for Step 1 because then it's easier to create pull requests

### CODE REVIEW

**Review by lukasbestle — APPROVED** [review]

**Review by lukasbestle — CHANGES_REQUESTED** [review]
Looks good. Could you please add information to the README about this?

**Comment by mxub** [comment]
do you want me to add the branch feature to the dev branch as well? And/Or add this functionality to other commands ?

**Comment by lukasbestle** [comment]
No, I just think that we need documentation for this new option.

**Comment by mxub** [comment]
yeah that definitely. What I meant was:

are there other commands that would benefit from the `--branch=dev`-option ?

**Comment by lukasbestle** [comment]
I think the plugin update command should have it as well. The other commands shouldn't need that.

**Comment by mxub** [comment]
i was just wondering if that could be a way to solve the version for kirby install

File: readme.md [hunk 1]
```diff
@@ -128,6 +128,14 @@ If you want to install a plugin from Kirby's official Plugins organisation (<htt
 kirby plugin:install cachebuster-plugin
 ```
 
+#### Branch Option
+
+You can pass a `--branch=dev`-Option to install the plugin form a specific branch. It defaults to the master branch.
```
Comment (lukasbestle) [inline readme.md]: Small typo: _from_ a specific branch. I also think we should make more clear that it works with any branch.

File: readme.md [hunk 2]
```diff
@@ -142,6 +150,14 @@ The shortcut for official plugins is working here as well:
 kirby plugin:update cachebuster-plugin
 ```
 
+#### Branch Option
+
+You can pass a --branch=dev-Option to install the plugin form a specific branch. It defaults to the master branch.
```
Comment (lukasbestle) [inline readme.md]: Same here. :)

**Review by lukasbestle — CHANGES_REQUESTED** [review]

**Comment by lukasbestle** [comment]
> i was just wondering if that could be a way to solve the version for kirby install

That command already has an option to select the develop branches. Other branches are only for development.

File: readme.md (hunk 1 above)
Comment (mxub) [reply readme.md]: is it clearer with 18120ba ?

**Review by lukasbestle — APPROVED** [review]

**Pull request closed without merging**